src/fparser/common/sourceinfo.py

```diff
@@ -67,17 +67,16 @@
 Provides functions to determine whether a piece of Fortran source is free or
 fixed format. It also tries to differentiate between strict and "pyf" although
 I'm not sure what that is.
-'''
 
-import io
+'''
```

Comment:
Remove offending import six and associated line that uses it (line 283)?

Reply:
Also fixed in #328  :)

Reply:
Great. For #328, we will wait.